src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -0,0 +1,35 @@
+PHPUnit Bridge
+==============
+
+Tools for helping with testing PHP projects, especially useful to deal with user
+deprecation notices. Should be used only in the `require-dev` section of your
+composer.json file (not in `require`).
+
+This bridge works only when used in conjunction with phpunit. It does:
```

Comment:
Actually, is we change the description to what I propose, I would simplify this sentence to just: `It provides the following features:`